### ISSUE TITLE
Shift to Collections of pages

### ISSUE THREAD

**Issue by vortexing** [FredHutch/wiki#6]
- [ ] Shift to the sidebars to be lists of collections of pages.
- [ ] Set up the yaml front matter to have:
- domain: [generation, informatics, computing]
- scope: [IRB and Human Subjects, etc]
- topic: [deidentification, etc)

**Comment by vortexing** [comment]
I am unable to do this.  I added a folder called curatedContent, and in it are three collection folders (_generation, _bioinformatics and _computing).  I added the collections to _config.yml, including defaults for pages in each, and the collection_dir to be curatedContent.  However I don't know how to access the rendered pages now and make them show up when you click a specific navbar tab.  

I watched this:
https://learn.cloudcannon.com/jekyll/introduction-to-jekyll-collections/
Looked at this example collections page:
https://github.com/github/choosealicense.com
Read this page about Jekyll Collections (specifically ):
https://jekyllrb.com/docs/collections/
And also this page, specifically Scenario 7:
https://jekyllrb.com/tutorials/navigation/

And I'm frustrated and not able to make it work.  I hope it's just Friday night and I just want to be done working.
